zookeeper-api/src/main/java/org/apache/helix/zookeeper/zkclient/ZkClient.java

```diff
@@ -1424,7 +1426,7 @@ public Boolean call() throws Exception {
     } finally {
       long endT = System.currentTimeMillis();
       if (LOG.isTraceEnabled()) {
-        LOG.trace("zkclient exists, path: {}, time: {} ms", _uid, path, (endT - startT));
+        LOG.trace("zkclient {} exists, path: {}, time: {} ms", _uid, path, (endT - startT));
```

Comment:
neat. thanks for fixing all logging bugs, I'm really curious do we even have any mechanism or test code path to verify this tracelog branches in code.

Reply:
I don't think we do, the IDE has some warnings though. And it's not hard failing after all.